mod/wet4/views/default/page/elements/head.php

```diff
@@ -259,16 +252,39 @@
 }
 
 // Meta tags for the page
+
+if (!$my_page_entity instanceof ElggEntity) {
+	$segments = substr($_SERVER['REQUEST_URI'], 0, strpos($_SERVER['REQUEST_URI'], '?'));
+	$segments = explode('/', $segments);
+	$some_post = get_entity($segments[sizeof($segments) - 1]);
+
+	if ($some_post instanceof ElggEntity) {
+		$my_page_entity = $some_post;
+		$page_entity_type = $my_page_entity->getSubtype();
+	} else {
+    $some_post = get_entity($segments[sizeof($segments) - 2]);
+  }
+	
+	if ($some_post instanceof ElggEntity) {
```

Comment:
Doubt it would make all that much difference but I'd put this inside the else above to prevent it from executing twice for the other cases.

Reply:
modified